client/src/ui/organisms/header/index.scss

```diff
@@ -45,6 +45,50 @@
   @media #{$mq-large-desktop-and-up} {
     flex-flow: row;
   }
+
+  #progress-bar {
```

Comment:
I'm not sure, but if this code is copied from material-ui, this might need some proper attribution, at least in the commit message, but possibly also in the code itself.

For this, it might make sense to extract the CSS to a dedicated `progress-bar.scss` and `@import` it instead.